src/app/about/About.module.css

```diff
@@ -0,0 +1,30 @@
+.about {
+    /* styles */
+    text-align: center;
+}
+
+.about__image_container {
+    display: flex;
+    justify-content: center;
+    margin-bottom: 2rem;
+    margin-top: 2rem;
+
+}
+
+.about__title {
+    margin-bottom: 5rem;
+    font-size: 3rem;
+    font-family: fantasy;
+}
+
+.about__description_container {
+    width: 100%;
+    
+}
+
+.about__description {
+    margin: 0 auto;
+    max-width: 1000px;
```

Comment:
[Comment]: I see the idea here. Normally, you would have some type of `container` class that would handle margin/padding for the page rather than apply it to specific elements (there's a few other tricks too for this to work better with mobile too). Anyways, keep this for now and we'll figure out a more general solution for all pages later as part of global styles.